CONTRIBUTING.md

```diff
@@ -9,11 +9,11 @@ Hello! In order for us (YANDEX LLC) to accept patches and other contributions fr
 
 By adopting the CLA, you state the following:
 
--   You obviously wish and are willingly licensing your contributions to us for our open source projects under the terms of the CLA,
--   You have read the terms and conditions of the CLA and agree with them in full,
--   You are legally able to provide and license your contributions as stated,
--   We may use your contributions for our open source projects and for any other our project too,
--   We rely on your assurances concerning the rights of third parties in relation to your contributions.
+- You obviously wish and are willingly licensing your contributions to us for our open source projects under the terms of the CLA,
```

Comment:
Same recommendation for CONTRIBUTING.md, LICENSE and AUTHORS - this time for legal reasons: we received those files as is and don't want to argue with lawyers.

Reply:
LICENSE and AUTHORS prettier does not process. `CONTRIBUTING.md` is handwritten, why not format it?